From f061100ad969742aef328f4d1bbcac87d664ba06 Mon Sep 17 00:00:00 2001
From: Alex Deucher <alexander.deucher@amd.com>
Date: Fri, 8 Nov 2019 11:22:57 -0500
Subject: [PATCH 4652/4736] drm/amdgpu/display: fix warning when
 CONFIG_DRM_AMD_DC_DCN is not set

dm_dcn_crtc_high_irq() is only used when CONFIG_DRM_AMD_DC_DCN is set.

Change-Id: I0a986136adc5fde5e6636de458580fc6080a1b9d
Reviewed-by: Nicholas Kazlauskas <nicholas.kazlauskas@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c
index f98ff5f012d2..bc76d49187fd 100755
--- a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c
+++ b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c
@@ -484,7 +484,7 @@ static void dm_crtc_high_irq(void *interrupt_params)
 	}
 }
 
-
+#if defined(CONFIG_DRM_AMD_DC_DCN1_0)
 /**
  * dm_dcn_crtc_high_irq() - Handles VStartup interrupt for DCN generation ASICs
  * @interrupt params - interrupt parameters
@@ -546,6 +546,7 @@ static void dm_dcn_crtc_high_irq(void *interrupt_params)
 
 	spin_unlock_irqrestore(&adev->ddev->event_lock, flags);
 }
+#endif
 
 static int dm_set_clockgating_state(void *handle,
 		  enum amd_clockgating_state state)
-- 
2.17.1

